compiler/fm-equalize/requires.cmake

```diff
@@ -1,2 +1,3 @@
+require("common-artifacts")
```

Comment:
Just for note, this is a temporal fix. If we have to include `fm-equalize` to the preset, we would need to just split the unittest from this folder or find other approaches because `common-artifacts` is not built in the preset.